components/calendar/src/any_calendar.rs

```diff
@@ -739,6 +861,10 @@ impl AnyCalendarKind {
             b"chinese" => AnyCalendarKind::Chinese,
             b"coptic" => AnyCalendarKind::Coptic,
             b"iso" => AnyCalendarKind::Iso,
+            b"islamic" => AnyCalendarKind::IslamicObservational,
+            b"islamicc" => AnyCalendarKind::IslamicCivil,
+            b"umalqura" => AnyCalendarKind::UmmAlQura,
```

Comment:
i think here is the most important spot because this is where we will parse the `u-ca` tag

Reply:
done

Reply:
I don't know if this code is reachable. When debugging with Andrew last week it seems the actual match statement which ICU4X uses is the one in `get_for_bcp47_value` a few paragraphs below this.